assets/src/application/ui/display/TimezoneTimeInfo/index.tsx

```diff
@@ -33,19 +33,14 @@ const TimezoneTimeInfo: React.FC<Props> = ({ date, ...props }) => {
 			<div className={'ee-focus-priority-6'}>{formatUtcDateForSite(date)}</div>
 		</div>
 	);
-	const className = classNames(props.className, 'ee-timezone-info');
-	const label = __('click for timezone\ninformation');
+	const className = classNames(props.className, 'ee-timezone-info', { 'ee-timezone-info--inline': inline });
```

Comment:
what's the purpose of the inline class?

are the popovers not anchored to the trigger location?

Reply:
I have no idea what is the purpose of that since it was added before here: https://github.com/eventespresso/event-espresso-core/pull/2605/files#diff-aef1c490c9f7e92212dead0972b64293R36

Reply:
set inline-block by default, to simplify things and it looks ok in both cases:

![image](https://user-images.githubusercontent.com/1477453/79210528-aa1f9700-7e4d-11ea-8ce8-194483b5d1b8.png)
***
![image](https://user-images.githubusercontent.com/1477453/79210562-b73c8600-7e4d-11ea-83ee-6558055a9899.png)

Reply:
oh I added that ?!?! lolz
must have been a prop that needed to be passed to an AntD component

Reply:
yes, cleared that out since now it's not needed/ not breaking anything